build/tasks/process.js

```diff
@@ -30,6 +30,12 @@ module.exports = function (grunt) {
       proc.kill('SIGINT');
     } else if (!proc) {
       procs[this.target] = spawn('node', [filePath]);
+
+      if (grunt.verbose) {
+        var logStream = fs.createWriteStream('services.log', {flags: 'a'});
+        procs[this.target].stdout.pipe(logStream);
+        procs[this.target].stderr.pipe(logStream);
+      }
```

Comment:
Hmmm. Interesting. I am not very opinionated here but want to pose a few questions.

Is it odd that it is tied to `grunt.verbose` but it doesn't actually write out to the console?
Is there any argument to be made to instead utilize a custom command line argument?

... maybe not.